schema/admin_schema.py

```diff
@@ -0,0 +1,9 @@
+from schema.base_schema import BaseSchema
+from datetime import datetime
+
+
+class AdminSchema(BaseSchema):
+    id: int
+    task_status_id: int
+    created_at: datetime
+    updated_at: datetime
```

Comment:
If you already have a schema with id you can use it as a parent schema instead of using `BaseSchema`